caffe2/CMakeLists.txt

```
@@ -969,6 +972,18 @@ if(USE_TBB)
 endif()
 
 
+if(NOT MSVC)
```

I guess you want to disable the functionality on Windows platform, rather than specifically for VisualC++ compiler?
```suggestion
if(NOT WIN32)
```